src/UiPath.Workflow.Runtime/ExtensionRequiredException.cs

```diff
@@ -0,0 +1,42 @@
+﻿// This file is part of Core WF which is licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace System.Activities;
+
+[Serializable]
+public class ExtensionRequiredException : Exception
+{
+    private const string RequiredExtensionTypeName = "requiredExtensionType";
+
+    public string RequiredExtensionTypeFullName { get; }
```

Comment:
if you fwd this to the Data member you won't need to handle serialization as it is already handled

Reply:
Data[RequiredExtensionTypeName]

Reply:
Done